src/main/kotlin/com/learnspigot/bot/manager/ForumManager.kt

```diff
@@ -98,11 +101,13 @@ class ForumManager(private val bot: JDA, private val datastore: Datastore, priva
                 valueRange = 0..25,
                 options = contributors.map { SelectOption.of(it.member.effectiveName, it.id) }
             )).complete()
+            removeOnClose.add(selectEmbed)
         }
-       channel.sendMessageEmbeds(Embed {
+       val hintEmbed = channel.sendMessageEmbeds(Embed {
```

Comment:
this could probably be done with a [scoped method](https://kotlinlang.org/docs/scope-functions.html#let) instead of making it a variable?
So after the .complete() you'd do
```kt
.let { removeOnClose.add(it) }
```
Same for the other cases where you create a variable for this.

Reply:
Ahh you are such a smart man, will fix this when I'm at pc

Reply:
Now done